### PR TITLE
Added ui-proxy to designationBulkUpload

### PR DESCRIPTION
**Pull request by ruksana2808**

### CODE REVIEW

File: src/proxies_v8/proxies_v8.ts [hunk 1]
```diff
@@ -961,6 +961,10 @@ proxiesV8.use('/v1/search/competenciesByOrg',
   proxyCreatorSunbird(express.Router(), `${CONSTANTS.KONG_API_BASE}`)
 )
 
+proxiesV8.use('/designationsUpload/*',
```
Comment (karthik-tarento) [inline src/proxies_v8/proxies_v8.ts]: Update the URL to -> /designation/*

File: src/utils/whitelistApis.ts [hunk 2]
```diff
@@ -3567,6 +3567,13 @@ export const API_LIST = {
                         ROLE.PUBLIC,
                    ],
         },
+        '/proxies/v8/designationsUpload/upload': {
```
Comment (karthik-tarento) [inline src/utils/whitelistApis.ts]: Update the URL to -> '/proxies/v8/designation/upload'
use the same in the below

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/proxies_v8/proxies_v8.ts (hunk 1 above)
Comment (ruksana2808) [reply src/proxies_v8/proxies_v8.ts]: made the change

File: src/utils/whitelistApis.ts (hunk 2 above)
Comment (ruksana2808) [reply src/utils/whitelistApis.ts]: made the changes

**Pull request merged**